src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/domain/BlogPost.java

```diff
@@ -95,4 +95,73 @@ public String getApprovalState() {
     public boolean isGoingInNewsletter(final LocalDateTime lastPublicationDate) {
         return publishedDate.isAfter(lastPublicationDate);
     }
+
+    public static BlogPostBuilder builder() {
```

Comment:
Is this builder really required? It's not used anywhere so it can be remove IMHO.